### DISCUSSION TITLE
LENS SONY 4-5.6/28-60

### DISCUSSION THREAD

**Discussion by marc94510** [lensfun/lensfun#2021] [Q&A]
Bonjour à tous 
J'aurais bien aim"é avoir le correctif lenfun pour  la lentille 4-5.6/28-60 et le boitiers SONY A7C mais je ne le trouve pas ou peut être qu'il n'est pas encore disponible. Si vous pouvez me donner des infos merci pour votre aide.

**Comment by sboukortt** [comment]
Bonjour Marc, merci pour ton intérêt. Il n’y a effectivement pas encore de données de correction pour cet objectif, mais si tu te sens motivé, tu peux essayer de les fournir en suivant le guide à cette adresse : https://pixls.us/articles/create-lens-calibration-data-for-lensfun/ ou bien juste prendre les photos nécessaires et nous laisser nous occuper de les analyser.
